source/Engine.cpp

```diff
@@ -2712,6 +2743,112 @@ void Engine::DoGrudge(const shared_ptr<Ship> &target, const Government *attacker
 
 
 
+size_t Engine::FleetPlacementLimit(const LimitedEvents<Fleet> &fleet, unsigned frames, bool requireGovernment)
```

Comment:
Why does this return a `size_t` if in a few cases it's restricted to `numeric_limits<int>:max()`?